Neos.ContentRepository.Core/Classes/Infrastructure/DbalSchemaFactory.php

```diff
@@ -48,9 +53,8 @@ public static function columnForNodeAggregateId(string $columnName): Column
      */
     public static function columnForContentStreamId(string $columnName): Column
     {
-        return (new Column($columnName, Type::getType(Types::STRING)))
-            ->setLength(36)
-            ->setCustomSchemaOption('charset', 'binary');
+        return (new Column($columnName, Type::getType(Types::BINARY)))
+            ->setLength(36);
```

Comment:
This change is not directly related but a follow-up to #4730 – without that change, `setup()` will always drop and re-add the column (and the status will reflect that)